argo-workflow/argo-install.yaml

```diff
@@ -289,10 +293,16 @@ spec:
         - --configmap
         - workflow-controller-configmap
         - --executor-image
-        - argoproj/argoexec:v2.11.7
+        - argoproj/argoexec:v3.0.0-rc3
         - --namespaced
         command:
         - workflow-controller
+        env:
+        - name: LEADER_ELECTION_IDENTITY
```

Comment:
What's this?

Reply:
you can have more controllers and this env identifies the leader.